x-pack/plugins/monitoring/server/kibana_monitoring/start_collector.js

```diff
@@ -53,4 +53,6 @@ export function startCollector(kbnServer, server, client, _sendBulkPayload = sen
   server.plugins.elasticsearch.status.on('red', () => {
     collector.cleanup();
   });
+
+  return collector;
```

Comment:
add `@return {TypeCollector}` to jsdoc?

Reply:
`TypeCollector` will get renamed to `CollectorSet` in my next PR, so I'll handle it there.

`TypeCollector` seemed like a bad name because it is a thing that keeps a set of collectors, not a collector itself.

Reply:
9c95f71148c7190eb679184eb5b2ed30695dece5